nixos/modules/services/x11/xserver.nix

```diff
@@ -14,9 +14,6 @@ let
   # Map video driver names to driver packages. FIXME: move into card-specific modules.
   knownVideoDrivers = {
     virtualbox = { modules = [ kernelPackages.virtualboxGuestAdditions ]; driverName = "vboxvideo"; };
-    ati = { modules = with pkgs.xorg; [ xf86videoati ]; };
-    intel = { modules = with pkgs.xorg; [ xf86videointel ]; };
-    modesetting = { modules = []; };
```

Comment:
About the last line: c7d92f94 confuses me but I'd expect your change to work OK.